ibis/pandas/core.py

```diff
@@ -302,6 +361,8 @@ def main_execute(expr, params=None, scope=None, aggcontext=None, **kwargs):
         The data that an unbound parameter in `expr` maps to
     scope : Mapping[ibis.expr.operations.Node, object]
         Additional scope, mapping ibis operations to data
+    timecontext : Optional[Tuple[pd.Timestamp, pd.Timestamp]]
+        timecontext needed for execution
```

Comment:
Can you add a link to doc and definition of timecontext to the API entry?

Reply:
I feel we are missing the user doc of how to use timecontext, for example, we should mention that it assumes there is a column called "time" across the entire expression tree, i.e., if you drop the time column, it will blow up.

Reply:
Discussed offline and decide to add user docs under `ibis.expr.types.Expr.execute`

Reply:
Discussed offline, one concern is that some backend doesn't support `kwargs`, `omniscidb` as an example. I just add a special case in omniscidbfor this `timecontext` param for now. We will continue to improve this once time context is applied to more backends.

Reply:
See the comments in omniscidb

Reply:
see my comments above, that backend needs fixing for this as its not obeying the standard signature

Reply:
Commented in another thread, I have changed the implementation to fix this in omniscidb backend.

Reply:
SGTM